spyder/plugins/ipythonconsole.py

```diff
@@ -861,12 +861,19 @@ def create_new_client(self, give_focus=True):
         """Create a new client"""
         self.master_clients += 1
         name = "%d/A" % self.master_clients
+        cf = self._new_connection_file()
+        if cf is None:
+            QMessageBox.warning(self, _('Warning'),

```

Comment:
Please don't use `QMessageBox.warning` for this message, but instead `client.show_kernel_error`, as it's done some lines below. This has the advantage of showing the message inside IPython consoles, and so it won't be easy to dismiss by our users (something that usually happens with modal dialogs as `QMessageBox`).

Also, when you do this change, please remove the `<br>`'s you added at the end of every line. We don't need them in this case :-)